healthtools_ke_api/__init__.py

```diff
@@ -52,7 +52,7 @@ def handle_error(error):
     try:
         response['description'] = error.description
     except Exception as err:
-        print err
+        print (err)
```

Comment:
I like that we are going Python3 standard here. There shouldn't be a space between `print` and `(` though.